48.md

```diff
@@ -0,0 +1,60 @@
+NIP-48
+======
+
+Proxy Tags
+----------
+
+`draft` `optional` `author:alexgleason`
+
+Nostr events bridged from other protocols such as ActivityPub can link back to the source object by including a `"proxy"` tag, in the form:
+
+```
+["proxy", <id>, <protocol>]
+```
+
+Where:
+
+- `<id>` is the ID of the source object. The ID format varies depending on the protocol. The ID must be universally unique, regardless of the protocol.
+- `<protocol>` is the name of the protocol, e.g. `"activitypub"`.
+
+Clients may use this information to reconcile duplicated content bridged from other protocols, or to display a link to the source object.
+
+Proxy tags may be added to any event kind, and doing so indicates that the event did not originate on the Nostr protocol, and instead originated elsewhere on the web.
+
+### Supported protocols
+
+This list may be extended in the future.
+
+| Protocol | ID format | Example |
+| -------- | --------- | ------- |
+| `activitypub` | URL | `https://gleasonator.com/objects/9f524868-c1a0-4ee7-ad51-aaa23d68b526` |
+| `atproto` | AT URI | `at://did:plc:zhbjlbmir5dganqhueg7y4i3/app.bsky.feed.post/3jt5hlibeol2i` |
+| `rss` | URL with guid fragment | `https://soapbox.pub/rss/feed.xml#https%3A%2F%2Fsoapbox.pub%2Fblog%2Fmostr-fediverse-nostr-bridge` |
+| `web` | URL | `https://twitter.com/jack/status/20` |
+
+### Examples
+
+ActivityPub object:
+
+```json
+{
+  "kind": 1,
+  "content": "I'm vegan btw",
```

Comment:
😂

Reply:
A bit late to the party but I'd love just to get clarification on one thing @alexgleason  ...

In this example, it's a `kind 1` event that proxies to another event on `activitypub`. The content of the activity pub event is completely different from the content on the nostr event. Are these proxy events meant to be mirrors of the events on other platforms or is this just a more structured way to link to things that are outside nostr?